test/routes/api/resources.js

```diff
@@ -0,0 +1,120 @@
+const assert = require('assert');
+const HttpStatus = require('http-status-codes');
+const _ = require('lodash');
+const session = require('supertest-session');
+
+const helper = require('../../helper');
+const app = require('../../../app');
+const models = require('../../../models');
+
+describe('/api/resources', () => {
+  let testSession;
```

Comment:
This is an _automated_ test, also called an "integration test" for the Resources API. See if you can understand what it's testing below...

This is an example of how you can test API endpoints even before you build the client-side forms and components that read/write from them.

A "back-end" engineer (not full-stack, not writing front-end) would verify that their code is correct by writing these types of automated tests... even if you're a full-stack engineer, having these tests is a way to ensure that the code works (and doesn't break as things change).